src/helpers/helpers.js

```diff
@@ -7,53 +7,73 @@ const writeFile = (path, data) => {
             else resolve();
         });
     });
-}
+};
 
 // const writeFile = (path, data) => new Promise((resolve, reject) => {
 //     fs.writeFile(path, data, err => {
 //         err ? reject(err) : resolve()
 //     })
 // })
 
-const readFile = (path) => {
+const readFile = path => {
     return new Promise((resolve, reject) => {
         fs.readFile(path, (error, data) => {
             if (error) reject(error);
             else resolve(data);
-        })
-    })
-}
+        });
+    });
+};
+
+const parseJSON = data => {
```

Comment:
Can someone provide an explanation for why is this promisified wrapper over the `JSON.parse` function necessary?